### PR TITLE
chore: generated CHANGELOG for origin/release/v52.6.0

### PR DESCRIPTION
**Pull request by randi274**
### What does this PR do?

Changelog for v52.6.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+
+## Fixed
+
+#### salesforcedx-sobjects-faux-generator
+
+- Correct LWC typing generation for new projects ([PR #3418](https://github.com/forcedotcom/salesforcedx-vscode/pull/3418))
+
+#### salesforcedx-vscode-core
+
+- Pr/3282 Use @heroku/functions-core library implementation of 'Start Function' and 'Run Function' ([PR #3399](https://github.com/forcedotcom/salesforcedx-vscode/pull/3399))
```
Comment (rcoringrato-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:21]: Move this to "Added":
```suggestion
- Performance improvements for  `SFDX: Start Function`, `Invoke` and `Debug Invoke` commands ([PR #3399](https://github.com/forcedotcom/salesforcedx-vscode/pull/3399))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+
+- Retrieve nothing should not error ([PR #3376](https://github.com/forcedotcom/salesforcedx-vscode/pull/3376))
```
Comment (rcoringrato-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:23]:
```suggestion
- Conflict detection no longer displays an error for empty results ([PR #3376](https://github.com/forcedotcom/salesforcedx-vscode/pull/3376))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+
+## Added
+
+#### salesforcedx-vscode
+
+- Include SOQL in extension pack ([PR #3400](https://github.com/forcedotcom/salesforcedx-vscode/pull/3400))
+
+#### salesforcedx-vscode-core
+
+- Update conflict detection setting ([PR #3416](https://github.com/forcedotcom/salesforcedx-vscode/pull/3416))
```
Comment (jag-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:11]:
```suggestion
- Update conflict detection setting description ([PR #3416](https://github.com/forcedotcom/salesforcedx-vscode/pull/3416))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,27 @@
...
+
+#### salesforcedx-sobjects-faux-generator
+
+- Correct LWC typing generation for new projects ([PR #3418](https://github.com/forcedotcom/salesforcedx-vscode/pull/3418))
+
+#### salesforcedx-vscode-core
+
+- Performance improvements for  `SFDX: Start Function`, `Invoke` and `Debug Invoke` commands ([PR #3399](https://github.com/forcedotcom/salesforcedx-vscode/pull/3399))
+
+- Conflict detection no longer displays an error for empty results ([PR #3376](https://github.com/forcedotcom/salesforcedx-vscode/pull/3376))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:23]:
```suggestion
- Fix conflict detection error for empty results ([PR #3376](https://github.com/forcedotcom/salesforcedx-vscode/pull/3376))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,29 @@
...
+
+#### salesforcedx-vscode
+
+- Include SOQL in extension pack ([PR #3400](https://github.com/forcedotcom/salesforcedx-vscode/pull/3400))
+
+#### salesforcedx-vscode-core
+
+- Update conflict detection setting description ([PR #3416](https://github.com/forcedotcom/salesforcedx-vscode/pull/3416))
+
+- Fix functions dependency issue preventing extensions from loading ([PR #3443](https://github.com/forcedotcom/salesforcedx-vscode/pull/3443))
```
Comment (jag-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]: Can we move this to fixed section?

**Review by jag-sfdc — APPROVED** [review]

**Pull request merged**